app/views.py

```diff
@@ -81,13 +89,58 @@ def progressreport():
     """Renders the report page."""
     return render_template('report.html', title=' | Progress Report')
 
-@app.route('/create', methods=['GET', 'POST'])
-def create():
-    """Renders the create page for educators."""
-    form = CreateQnForm()
-    return render_template('create.html', title=' | Create', form=form)
-
-
+@app.route('/createclass', methods=['GET', 'POST'])
+def createclass():
+    """Renders the create class page for educators."""
+    form = CreateClass()
+    if form.validate_on_submit():
+        group = Group(name=form.className.data)
+        db.session.add(group)
+        classCode = set_code(6)
+        db.session.add(classCode)
+        db.session.commit()
+        return redirect(url_for('createclasssuccess'))
+    return render_template('createclass.html', title=' | Create Class', form=form)
+
+@app.route('/createclasssuccess', methods=['GET'])
+def createclasssuccess():
+    """Renders the create class was a success page for educators."""
+    return render_template('createclasssuccess.html', title=' | Create Class', form=form)
+
+@app.route('/createquiz', methods=['GET', 'POST'])
+def createquiz():
+    """Renders the create quiz page for educators."""
+    form = CreateQuiz()
+    return render_template('createquiz.html', title=' | Create Quiz', form=form)
+
+@app.route('/createnewquiz', methods=['POST'])
+def createnewquiz():
+    """Routing for creating a new quiz."""
+    form = CreateQuiz()
+    #if form.validate_on_submit():
+        # Commit inputs to database
+    return render_template('createquiz.html', title=' | Create Quiz', form=form)
+
+@app.route('/createquizsuccess', methods=['GET'])
+def createquizsuccess():
+    """Renders the create quiz was a success page for educators."""
+    return render_template('createquizsuccess.html', title=' | Create Quiz', form=form)
+
+@app.route('/class', methods=['GET'])
+def classes():
+    """Renders the class page."""
+    image_file = url_for('static', filename='resources/images/profile_pics/' + current_user.image_file)
+    return render_template('sidebar.html', image_file=image_file, title=' | Class')
+
+@app.route('/class/newID')
+def update_class_code():
```

Comment:
```suggestion
@app.route('/class/<int:groupID>')
def update_class_code(groupID):
```
I can do this part, but this is how the class will be accessed via routing.

Reply:
noted, will update